src/namedentity.cpp

```diff
@@ -19,44 +19,41 @@ limitations under the License.
 #include "libcellml/component.h"
 #include "libcellml/componententity.h"
 
+#include "namedentity_p.h"
+
 namespace libcellml {
 
-/**
- * @brief The NamedEntity::NamedEntityImpl struct.
- *
- * The private implementation for the NamedEntity class.
- */
-struct NamedEntity::NamedEntityImpl
+inline NamedEntity::NamedEntityImpl *NamedEntity::pFunc()
```

Comment:
Remove `inline` (and do the same in all the other places where we use `inline`, at least those in this PR).

Reply:
a7f9eb8, df859a2